ui/component/fileViewerInitiator/view.jsx

```diff
@@ -48,20 +50,29 @@ export default function FileViewerInitiator(props: Props) {
     hasCostInfo,
     costInfo,
     isAutoPlayable,
+    claim,
   } = props;
   const cost = costInfo && costInfo.cost;
   const forceVideo = ['application/x-ext-mkv', 'video/x-matroska'].includes(contentType);
   const isPlayable = ['audio', 'video'].includes(mediaType) || forceVideo;
   const fileStatus = fileInfo && fileInfo.status;
   const webStreamOnly = contentType === 'application/pdf' || mediaType === 'text';
   const supported = IS_WEB ? (!cost && isStreamable) || webStreamOnly || forceVideo : true;
+  const { name, claim_id: claimId, value } = claim;
+  const fileName = value && value.source && value.source.name;
+  const downloadUrl = `/$/download/${name}/${claimId}`;
 
-  function getMessage() {
-    if (IS_WEB && cost) {
-      return __('Paid Content Not Supported on lbry.tv');
+  function getTitle() {
```

Comment:
I think the messaging and presentation of this should switch more strongly. At this point if it can't be rendered there is a good chance downloading is simply intended.